crates/turborepo-lib/src/commands/ls.rs

```diff
@@ -99,21 +170,44 @@ impl<'a> RepositoryDetails<'a> {
             packages,
         }
     }
-    fn print(&self) -> Result<(), cli::Error> {
-        if self.packages.len() == 1 {
-            cprintln!(self.color_config, BOLD, "{} package\n", self.packages.len());
-        } else {
-            cprintln!(
-                self.color_config,
-                BOLD,
-                "{} packages\n",
-                self.packages.len()
-            );
-        }
+    fn pretty_print(&self) {
+        let package_copy = match self.packages.len() {
+            0 => "no packages",
+            1 => "package",
+            _ => "packages",
+        };
+
+        cprint!(
+            self.color_config,
+            BOLD,
+            "{} {} ",
+            self.packages.len(),
+            package_copy
+        );
+        cprintln!(self.color_config, GREY, "({})\n", self.package_manager);
 
         for (package_name, entry) in &self.packages {
             println!("  {} {}", package_name, GREY.apply_to(entry));
         }
+    }
+
+    fn json_print(&self) {
+        let as_json = serde_json::to_string_pretty(&self);
+        match as_json {
+            Ok(json) => println!("{}", json),
+            Err(err) => println!(r#"{{"error": "{}"}}"#, err),
```

Comment:
I would recommend against pushing an error into the JSON and instead throw the error. As of https://github.com/vercel/turborepo/pull/8969 we now print errors to `stderr` so piping this to `jq` shouldn't cause issues.

Putting the error into JSON leads to a confusing `jq` error:
```
$ echo '{"error": "bad JSON"}' | jq '.packages | map(.path)'
jq: error (at <stdin>:1): Cannot iterate over null (null)
```